prusti-server/src/client.rs

```diff
@@ -30,6 +30,27 @@ impl PrustiClient {
     pub async fn verify(
         &self,
         request: VerificationRequest,
+    ) -> reqwest::Result<VerificationResult> {
+        let response = if let VerificationBackend::Lithium = request.backend_config.backend {
+            self.verify_natively(request).await
```

Comment:
Instead of branching on the client side, I would suggest branching on the server side because this allows doing (at least theoretically) the heavy verification task on a different machine.